app/com/arpnetworking/metrics/portal/alerts/impl/DatabaseAlertExecutionRepository.java

```diff
@@ -92,13 +124,26 @@ private AlertExecution findOrCreateAlertExecution(
     public void open() {
         assertIsOpen(false);
         LOGGER.debug().setMessage("Opening DatabaseAlertExecutionRepository").log();
+        try {
+            _partitionCreator = _actorSystem.actorOf(_props);
+            DailyPartitionCreator.start(_partitionCreator, Duration.ofSeconds(5));
+        } catch (final ExecutionException | InterruptedException e) {
+            throw new RuntimeException("Failed to start partition creator", e);
+        }
         _isOpen.set(true);
     }
 
     @Override
     public void close() {
         assertIsOpen();
         LOGGER.debug().setMessage("Closing DatabaseAlertExecutionRepository").log();
+        assert _partitionCreator != null : "partitionCreator should be non-null when open";
```

Comment:
We generally avoid the use of assert since it's conditional based on JVM arguments